samples/emoji-search/ios/swiftUI/EmojiSearchApp/Widget/ColumnBinding.swift

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import shared
+import SwiftUI
+
+final class ColumnBinding: BaseWidget, WidgetColumn, SwiftUIViewBinding {
+    func height(height: Int32) {
+        
+    }
+
+    func horizontalAlignment(horizontalAlignment: Int32) {
```

Comment:
@skorulis-ap The way we've been implementing these params to far is to pass them to a [layout engine](https://github.com/cashapp/redwood/blob/trunk/redwood-layout-uiview/src/commonMain/kotlin/app/cash/redwood/layout/uiview/UIViewFlexContainer.kt#L42) written in common Kotlin code. Basically it takes size constraints + views and returns positions for each of the children.

Is it possible to position children manually in a `VStack` or `HStack`? Else, we'll probably have to do something custom for Swift UI.

Reply:
Manual positioning is possible, but kind of fights the system. That I think is less of a worry than getting the sizes of each child. SwiftUI calculates the sizes during the layout pass so you would need to have some sort of side rendering happening to get these values. 
iOS 16 adds a [Layout](https://developer.apple.com/documentation/swiftui/layout) object but that's not an option for a few years.

My plan was to use the SwiftUI layout system and respect the parameters that are passed in. Overflow likely being the hardest one to implement.